crates/core/protobuf/node_ctl_svc.proto

```diff
@@ -27,6 +27,9 @@ service NodeCtlSvc {
 
   // Returns the cluster health from the point of view of this node.
   rpc ClusterHealth(google.protobuf.Empty) returns (ClusterHealthResponse);
+
+  // Try to remove the given node from the cluster.
+  rpc RemoveNode(RemoveNodeRequest) returns (google.protobuf.Empty);
```

Comment:
Should this be moved to admin API?

Reply:
Alternatively, maybe we can move the entire logic to restatectl and only introduce the RPC API when we have maintenance workflows.

Reply:
Moving it to `restatectl` should now be possible since we no longer need access to `Bifrost` for finding the trim-points. Maybe this is then the simplest thing to do until we have the workflow. Will update the PR.